src/components/Tags/Tags.tsx

```diff
@@ -16,32 +21,28 @@ export interface TagsListProps {
    */
   className?: string;
   /**
-   * Individual Tag class
+   * Child element pass in to display as item content.
    */
-  tagClassName?: string;
+  children?: React.ReactNode;
   /**
-   * List of tags to render
-   */
-  tagsList: string[];
-  /**
-   * `onRemove` callback removes the clicked tag from the list
+   * Individual Tag class
    */
-  onRemove: (tag: string) => void;
+  tagClassName?: string;
   /**
    * `onRemove` callback provides the tag that is to be removed as a string
    */
   onClear: () => void;
+  /**
+   * Clear All label is translatable
+   */
+  i18n?: I18nObject;
```

Comment:
I've seen this pattern before, but we haven't been doing this in other components, instead just saying `clearAllLabel`